packages/x-svg-converter/src/__tests__/svg-to-vue.spec.ts

```diff
@@ -0,0 +1,102 @@
+import path from 'path';
+import fs from 'fs';
+import { svgToVue } from '../svg-to-vue';
+import svgStub from './svg-stub';
+
+// Mock to prevent the prettier from running in test env.
+jest.mock('child_process', () => {
+  const originalModule = jest.requireActual('child_process');
+
+  return {
+    __esModule: true,
+    ...originalModule,
+    exec: jest.fn(() => true)
+  };
+});
+
+describe('test SVG to Vue script', () => {
+  const sourcePath = './src/__tests__/svgs';
+
+  afterEach(() => {
+    jest.resetAllMocks();
+
+    generateTestSVGs();
+    removeVueTestFiles();
+  });
+
+  it('should create a vue component for each svg in the source folder', () => {
+    process.argv = ['param1', 'param2', sourcePath];
```

Comment:
What do you think about extracting this to a helper function such as `setScriptArgs` or something like that, and pass to that function only the arguments that go after the first two process args?

I think it can make the test a bit clearer.

Reply:
In the case of these tests, we get the same result by setting the parameters in a `beforeEach` and overriding them when needed (is needed in 1 test).